js/src/wp-seo-admin-global.js

```diff
@@ -128,3 +128,47 @@
 	window.wpseoMakeDismissible = wpseoMakeDismissible;
 	window.wpseoDismissLink = wpseoDismissLink;
 }());
+
+(function( $ ) {
+	'use strict';
+
+	$( '.nav-tab' ).click( function() {
+		closeVideoSlideout( 0 );
+	} );
+
+	$( '.wpseo-tab-video-container' ).on( 'click', '.wpseo-tab-video-container__handle', function( e ) {
+		var $target = $( e.delegateTarget ).find( '.wpseo-tab-video-slideout' );
+		if ( $target.is( ':hidden' ) ) {
+			openVideoSlideout( $target, 400 );
+		}
+		else {
+			closeVideoSlideout( 400 );
+		}
+		return false;
```

Comment:
This way of cancelling the default action is not desirable because it stops the event bubbling.